modules/protobuf/27.0/presubmit.yml

```diff
@@ -0,0 +1,67 @@
+matrix:
+  platform: ["debian11", "macos", "ubuntu2004"]
+  bazel: [6.x, 7.x]
+
+tasks:
+  verify_targets:
+    name: "Verify build targets"
+    platform: ${{ platform }}
+    bazel: ${{ bazel }}
+    build_flags:
+    - '--host_cxxopt=-std=c++14'
+    - '--cxxopt=-std=c++14'
+    - '--host_copt=-Wno-deprecated-declarations'
+    - '--copt=-Wno-deprecated-declarations'
+    build_targets:
+    - '@protobuf//:protobuf'
+    - '@protobuf//:protobuf_lite'
+    - '@protobuf//:protoc'
+    - '@protobuf//:test_messages_proto2_cc_proto'
+    - '@protobuf//:test_messages_proto3_cc_proto'
+
+  verify_targets_windows:
+    name: "Verify build targets"
+    platform: windows
```

Comment:
Why was windows split out here?

Reply:
`-Wno-deprecated-declarations` isn't supported by MSVC but is required on debian with the protobuf version and compiler version running right now.